src/app_service/service/message/dto/seen_unseen_messages.nim

```diff
@@ -0,0 +1,16 @@
+import json
+
+include ../../../common/json_utils
+
+type SeenUnseenMessagesDto* = object
```

Comment:
I am not sure if it's a good practice to expose the attributes of the module. I know it's okay in some languages like Python. I believe it's good to follow encapsulation by leaving the attributes private. Is it common in the codebase to have those attributes as public?

Reply:
Basically the star`*` here says that type is public, a yes we are widely reusing types across modules